components/ILIAS/DataCollection/classes/Table/class.ilDclTableEditGUI.php

```diff
@@ -72,13 +76,15 @@ public function __construct(ilDclTableListGUI $a_parent_obj)
         }
     }
 
+    protected int $obj_id;
+
     public function executeCommand(): void
     {
         $cmd = $this->ctrl->getCmd();
 
         switch ($cmd) {
             case 'update':
-                $this->save("update");
+                $this->update();
```

Comment:
the case `update` can be removed since its identical to the default

Reply:
Done!